zcash_client_sqlite/src/chain.rs

```diff
@@ -139,6 +139,16 @@ pub(crate) fn blockmetadb_insert(
     }
 }
 
+#[cfg(feature = "unstable")]
+pub(crate) fn blockmetadb_rewind_to_height(
+    conn: &Connection,
+    block_height: BlockHeight,
+) -> Result<(), rusqlite::Error> {
+    conn.prepare("DELETE FROM compactblocks_meta WHERE height > ?")
```

Comment:
```suggestion
    conn.prepare("DELETE FROM compactblocks_meta WHERE height > ?")?
```

Reply:
Missed the question mark (to return the result as an error) rather than `unwrap`.